src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -161,26 +183,35 @@ def run_conversion(
         Returns
         -------
         NWBFile
-
         """
         file_paths = self.source_data["file_paths"]
         audio_metadata = metadata["Behavior"]["Audio"]
-        # Checks for metadata
-        _check_file_paths(file_paths=file_paths, metadata=audio_metadata)
         _check_audio_names_are_unique(metadata=audio_metadata)
+        assert len(audio_metadata) == self._number_of_audio_files, (
+            f"The Audio metadata is incomplete ({len(audio_metadata)} entry)! "
+            f"Expected {self._number_of_audio_files} (one for each entry of 'file_paths')."
+        )
 
         audio_name_list = [audio["name"] for audio in audio_metadata]
         any_duplicated_audio_names = len(set(audio_name_list)) < len(file_paths)
         if any_duplicated_audio_names:
             raise ValueError("There are duplicated file names in the metadata!")
 
-        starting_times = _check_starting_times(starting_times=starting_times, metadata=audio_metadata)
+        if self._number_of_audio_files > 1 and self._segment_starting_times is None:
+            raise ValueError(
+                "If you have multiple audio files, then you must specify each starting time by calling "
+                "'.align_segment_starting_times(segment_starting_times=...)'!"
+            )
+        starting_times = self._segment_starting_times or [0.0]
```

Comment:
Isn't this considred bad practice? Testing for None as a boolean? What is your take on that?

Reply:
I don't think it is exactly that. I think is a combination of two things:
1) https://en.m.wikipedia.org/wiki/Short-circuit_evaluation
2) How python defines truthiness of certain objects 

That's how the [documentation](https://docs.python.org/3/library/stdtypes.html#truth-value-testing) of python talks about it and therein likes the usual critique. The expression works because None evaluates to false and then x = None or y returns y. The problem is that for python other stuff like `0`, `[]` and `""` have values of False. So if you really wanted to test for None only you would have a problem.

Here it is not a problem because for 0, [], and None values of `self._segment_starting_times` it makes sense that they default to [0] as you do. But I was just wondering if you considered that, I think it is good to be aware of.

Reply:
> I think it is good to be aware of.

Fair enough


> But I was just wondering if you considered that

I did yeah, that's also why I like that behavior of Python. If any type of nullish states occur (`None` from not specified, empty from who knows what, or `0`/`0.0` but not wrapped in a list), use the fallback value; if not a nullish state then trust it was set correctly and checked by other assertions

Reply:
Thanks for sharing your view!